migrations/2024-10-28-125949_change_currency_type_in_dispute/up.sql

```diff
@@ -0,0 +1,2 @@
+-- Your SQL goes here
+ALTER TABLE dispute ALTER COLUMN currency TYPE "Currency" USING currency::"Currency"; -- Migration query to be run after deployment before running this query
```

Comment:
Won't this lock the entire table, and possibly affect deployments which happen in a staggered manner?